File: Java/Datatest.java
public class Datatest {
	public static void main(String[] args) {

        LinkedList linkedList = new LinkedList();
        System.out.println(linkedList.isEmpty());
        linkedList.insertLast(3);
        linkedList.insertLast(5);
        linkedList.insertLast(6);
        linkedList.insertLast(8);
        linkedList.insertLast(9);
        linkedList.insertFirst(1);
        linkedList.insertLast(233);
        System.out.println(linkedList.toString());
        linkedList.extract();
        System.out.println(linkedList.isEmpty());
        System.out.println(linkedList.search(233));
	}
	
}

File: C++/Datatest.cpp
#include <iostream>
#include "stdlib.h"
#include "stdio.h"
#include "LinkedList.h"
#include "LinkedList.cpp"

int main()
{
    LinkedList linkedList;
    linkedList.insertLast(30);
    linkedList.insertLast(10);
    linkedList.insertLast(5);
    linkedList.insertLast(12);
    linkedList.insertLast(25);
    linkedList.insertLast(250);
    linkedList.insertLast(500);
    linkedList.insertFirst(1);
    cout<<linkedList.isEmpty()<<endl;
    cout<<linkedList.search(12)<<endl;
    cout<<linkedList.search(2)<<endl;
    linkedList.toString();
    linkedList.clear();
    return 0;
}
